lib/internal/util/inspect.js

```diff
@@ -818,6 +794,46 @@ function getIteratorBraces(type, tag) {
   return [`[${tag}] {`, '}'];
 }
 
+function getBoxedBase(value, ctx, keys, constructor, tag) {
+  let fn;
+  let type;
+  if (isNumberObject(value)) {
+    fn = NumberPrototype;
+    type = 'Number';
+  } else if (isStringObject(value)) {
+    fn = StringPrototype;
+    type = 'String';
+    // For boxed Strings, we have to remove the 0-n indexed entries,
+    // since they just noisy up the output and are redundant
+    // Make boxed primitive Strings look like such
+    keys.splice(0, value.length);
+  } else if (isBooleanObject(value)) {
+    fn = BooleanPrototype;
+    type = 'Boolean';
+  } else if (isBigIntObject(value)) {
+    fn = BigIntPrototype;
+    type = 'BigInt';
+  } else {
+    fn = SymbolPrototype;
+    type = 'Symbol';
+  }
+  let base = `[${type}`;
+  if (type !== constructor) {
+    if (constructor === null) {
+      base += ' (null prototype)';
```

Comment:
nit: Not a blocking comment.. But my thoughts..For objects, null prototype is showed as `[Object: null prototype] {}`, looks like for primitives, we show as `[Boolean (null prototype): true]` (like in test), may be we can show like `[Boolean: null prototype] true`, so that reading is consistent.

Reply:
I thought about that as well but I think it would be less obvious that it's a boxed primitive in that case (obviously, e.g., `5` is a number, so it might also just stand for extra information?).

The inconsistency is not ideal but this is likely going to be a super rare case anyway.